README.md

```diff
@@ -62,6 +62,31 @@ toolchain that you want to use. You may have to pass `register.py` different
 arguments depending on the type of toolchain. See `register.py`'s `--help`
 text for more information.
 
+## Using the Docker Container
+
+This repository also includes a dockerfile which can be used to run a Jupyter Notebook instance which includes this Swift kernel. To build the container, the following command may be used:
+
+```bash
+# from inside the directory of this repository
+docker build -f docker/Dockerfile -t swift-jupyter .
+```
+
+The resulting container comes with the latest Swift for TensorFlow toolchain installed, along with Jupyter and the Swift kernel contained in this repository.
+
+This container can now be run with the following command:
+
+```bash
+docker run -p 8888:8888 --security-opt seccomp:unconfined -v /my/host/notebooks:/notebooks swift-jupyter
```

Comment:
Could you change it to `--cap-add SYS_PTRACE`? That works too, and it's a bit less privilege than `-security-opt seccomp:unconfined`